rs/nervous_system/root/src/change_canister.rs

```diff
@@ -64,7 +64,7 @@ pub struct ChangeCanisterRequest {
     #[serde(with = "serde_bytes")]
     pub wasm_module: Vec<u8>,
 
-    /// If the entire WASM does not into the 2 MiB ingress limit, then `new_canister_wasm`
+    /// If the entire WASM does not fit into the 2 MiB ingress limit, then `new_canister_wasm`
```

Comment:
I don't see a field called "new_canister_wasm" - should that say "wasm_module"?

Reply:
Indeed, good catch!